src/plugins/github/nodes.js

```diff
@@ -125,6 +127,12 @@ export function fromRaw(x: RawAddress): StructuredAddress {
   function fail() {
     return new Error(`Bad address: ${NodeAddress.toString(x)}`);
   }
+  if (NodeAddress.hasPrefix(x, GitNode.Prefix.commit)) {
```

Comment:
This dereference really concerns me. See #818. I won’t block this PR on
these grounds.

Reply:
Fixed per your suggestion.